aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1509,6 +1523,12 @@ impl AptosVM {
                 .is_enabled(FeatureFlag::SAFER_RESOURCE_GROUPS),
         )?;
         verifier::event_validation::validate_module_events(session, modules)?;
+        if self
+            .features()
+            .is_enabled(FeatureFlag::DISALLOW_USER_NATIVES)
+        {
+            verifier::native_validation::validate_module_natives(modules)?;
+        }
```

Comment:
Wonder if we should move this check to a slightly earlier point, considering how trivial it is. No string opinions though

Reply:
Done!